app/models/place.rb

```diff
@@ -29,4 +33,52 @@ def self.prefectures_with_key
       }
     end
   end
+
+  def self.create_index!(options={})
+    client = __elasticsearch__.client
+    settings index: {
+               analysis: {
+                 tokenizer: {
+                   ngram_tokenizer: {
+                     type: 'nGram',
+                     min_gram: '2',
+                     max_gram: '3',
+                     token_chars: ['letter', 'digit']
+                   }
+                 },                 
+                 analyzer: {
+                   ngram_analyzer: {
+                     tokenizer: "ngram_tokenizer"
+                   },
+                   kuromoji_analyzer: {
+                     tokenizer: 'kuromoji_tokenizer',
+                     filter:    ['kuromoji_baseform'],
+                   }
+                 }
+               }
+             } do
```

Comment:
__[rubocop]__
* Indent the right brace the same as the start of the line where the left brace is.